Fix grammar in HTML label element (#25901)
mdn/content: files/en-us/web/html/element/label/index.md

Associating a `<label>` with a form control, such as {{htmlelement("input")}} or {{htmlelement("textarea")}} offers some major advantages:

> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.

Reviewer: ```suggestion
- The label text is not only visually associated with its corresponding text input; they are programmatically associated, too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
```

- When a user clicks or touches/taps a label, the browser passes the focus to its associated input (the resulting event is also raised for the input). That increased hit area for focusing the input provides an advantage to anyone trying to activate it — including those using a touch-screen device.

> To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.

Reviewer 2: ```suggestion
To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, add the `for` attribute to the `<label>` element. Lastly, make the value of `for` the same as the `id` in the `<input>` element.
```

### Interactive content

> Do not place interactive elements such as {{HTMLElement("a", "anchors")}} or {{HTMLElement("button", "buttons")}} inside a `label`. Doing so makes it difficult to activate the form input associated with the `label`.

Reviewer: No need to expand contractions: they are generally encouraged in technical writing to make it sound casual.
